health_monitoring/test_health_monitor.py

```diff
@@ -404,3 +404,176 @@ def test(self):
         s = self.get_server("deproxy")
         s.drop_conn_when_receiving_data = True
         time.sleep(1)
+
+
+class TestHmCrcAutoImplicit(tester.TempestaTest):
+    """
+    Health monitor configuration called "auto" uses "auto" CRC mode.
+    In this mode CRC for comparation calculated on the fly from
+    the first server response.
+    Default HM configuration is "auto".
+    """
+
+    tempesta_implicit_auto = {
+        "config": """
+            listen 80;
+
+            server_failover_http 404 3 10;
+
+            srv_group main {
+            server ${server_ip}:8080;
+
+            health auto;
+            }
+    """
+    }
+
+    bk_dp = {
+        "id": "deproxy",
+        "type": "deproxy",
+        "port": "8080",
+        "response": "static",
+        "response_content": "HTTP/1.0 200 OK\r\nContent-Length:5\r\n\r\nHello",
+    }
+
+    tempesta = tempesta_implicit_auto
+    backends = [bk_dp]
+
+    def test(self):
+        klog = dmesg.DmesgFinder(disable_ratelimit=False)
```

Comment:
You can use `self.oops` here and `@dmesg.unlimited_rate_on_tempesta_node` decorator. (optional). You must add `self.addCleanup()` for `disable_ratelimit=False`. [Example](https://github.com/tempesta-tech/tempesta-test/blob/8dbaa46762a72a24e1985c4d77ef5d7e92d1f55e/sessions/test_js_challenge.py#L672). We have the tests where ratelimit must be a small.